modules/catalog/src/test/java/org/apache/ignite/internal/catalog/commands/AlterTableAlterColumnCommandValidationTest.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.ignite.internal.catalog.commands;
 
+import static org.apache.ignite.internal.catalog.CatalogManagerSelfTest.applyNecessaryPrecisionScale;
```

Comment:
we have discussed this already: if you need utility method, please move it to utility class